### DISCUSSION TITLE
Works on customized Linux flavors?

### DISCUSSION THREAD

**Discussion by XinyueZ** [fleaflet/flutter_map#1194] [Q&A] [answered]
Hey,

I just see that the lib supports Linux, too.

I am not sure if it works on standard Linux or even some customized Linux flavors?

<img width="588" alt="Bildschirmfoto 2022-03-22 um 16 21 50" src="https://user-images.githubusercontent.com/7869833/159517354-9a62e041-37a0-490b-8d84-10315c0547f9.png">

🤙

**Comment by ibrierley** [comment, marked as answer]
Git seems to be not enabling comments atm, so trying via email, see if this
displays..

It works on Mint so I assume it would be ok on Ubuntu. I'm also guessing it
would be ok on others, but mileage may vary. There may be some things that
aren't supported, for example if a plugin uses isolates it wouldn't work,
or maybe if it needs geolocation or something.

Basic steps would be something like...
```
apt install clang (if not already installed)
apt install libgtk-3-dev
flutter config --enable-linux-desktop
flutter create . (in your project folder)
(restart android studio if using that)
flutter run -d linux example/lib/main.dart (or you should see a Linux
desktop option to run in Android Studio)

```

**Reply by JaffaKetchup** [reply]
Hi @XinyueZ, has this answered your question? If so, please mark as answer.

**Reply by XinyueZ** [reply]
almost

**Comment by XinyueZ** [comment]
close
